cicd/gitops/monitoring-argo-cd-custom-resource-workloads.adoc

```diff
@@ -0,0 +1,38 @@
+:_content-type: ASSEMBLY
+[id="monitoring-argo-cd-custom-resource-workloads"]
+= Monitoring Argo CD custom resource workloads
+include::_attributes/common-attributes.adoc[]
+:context: monitoring-argo-cd-custom-resource-workloads
+
+toc::[]
+
+[role="_abstract"]
+With {gitops-title}, you can monitor the availability of Argo CD custom resource workloads for specific Argo CD instances. By monitoring Argo CD custom resource workloads, you have the latest information about the state of your Argo CD instances by enabling alerts for them. The Operator then triggers the alerts when the component workload pods such as application-controller, repo-server, or server of the correpsonding Argo CD instance are unable to come up for certain reasons and there is a drift between the number of ready replicas and the number of desired replicas for a certain period of time.
+
+You can enable and disable the setting for monitoring Argo CD custom resource workloads. 
+
+// Prerequisites for monitoring Argo CD custom resource workloads
+[discrete]
+== Prerequisites
+
+* {gitops-title} is installed in your cluster.
+* Logged into Argo CD instance.
```

Comment:
This should probably be a full sentence, to match the others, eg

You are logged into...

Reply:
@Srivaralakshmi On second thought, being logged into Argo CD instance is not really required, we can probably get rid of this point 
As long as users are logged into their OpenShift cluster that is enough for this feature

Reply:
@gabriel-rh @jaideepr97 Removed this point.